ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -222,8 +223,8 @@ class TFLiteCore
   accl_hw accelerator;
   tflite_delegate_e delegate;
 
-  TFLiteInterpreter interpreter;
-  TFLiteInterpreter interpreter_sub;
+  TFLiteInterpreter *interpreter;
```

Comment:
This might be a bit out of scope of this PR, If ownership is shared later on, deleting at destruction would be dangerous. How about using `std::shared_ptr` later on?

Reply:
I understand what you worry about.
I will consider the point deeply for better sharing representation.

Reply:
@helloahn 
Q: Is it shared with other instances? or let them own it?

Reply:
Then, in addition to using ```std::sharred_ptr```, how about applying the singleton pattern to it?

Reply:
IMO, it could be a solution.
Since we still need to maintain multiple interpreters respectively,
I'm planning to manage them with a hash table.
The singleton pattern could be used at this point.